content/blog/20250806_fluentd-v1.19.0-has-been-released.md

```diff
@@ -0,0 +1,122 @@
+# Fluentd v1.19.0 has been released
+
+Hi users!
+
+We have released v1.19.0 on 2025-07-30.
+ChangeLog is [here](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md#release-v1190---20250730).
+
+This release is a new release of v1.19 series.
+This release includes many new features and bug fixes, with a particular focus on reducing operational and running costs and improving overall stability.
+
+## Major new features and improvements
+### Add feature to evacuate chunk files when retry limit
+In this release, we added a buffer evacuation feature to the `buf_file` and `buf_file_single` plugins.
+This simplifies recovery after failure.
+
+When an output plugin fails to deliver data by network issue,
+in previous versions, if the retry limit was exceeded and secondary output was not configured, the data would be discarded.
+
```

Comment:
It might be natural "In the previous versions, ...." then continues with "In this release" wording.